distribution/docs/src/main/resources/content/_securing/hardening-intro.adoc

```diff
@@ -16,7 +16,11 @@ Where available, necessary migitations to harden an installation of ${branding}
 Refer to the <<{managing-prefix}hardening_checklist,Hardening Checklist>> for a compilation of these mitigations.
 ****
 
+[NOTE]
+====
 These security precautions are best performed as configuration is taking place, so hardening steps are integrated into configuration steps.
```

Comment:
What follows is a list of what should be configured for a hardened system. It is not intended to be a step-by-step procedure. To harden a new system, perform configuration as <<${managing-prefix}configuring.adoc,documented>>.  The configuration procedure includes instructions to hardening ${branding}.

Reply:
After looking at this together, @ahoffer and I agreed that this info may be more useful as an appendix.